packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -165,24 +279,30 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
       );
 
     final Color inactive = CupertinoDynamicColor.resolve(inactiveColor, context);
+
+    // TODO: allow dynamic resizing
+    final bool isIconCompact = (CupertinoSizeClassHelper.getWidthSizeClass(context) == CupertinoSizeClass.compact) && (CupertinoSizeClassHelper.getHeightSizeClass(context) == CupertinoSizeClass.compact);
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values

> Avoid using if chains, ? …​ : …​, or, in general, any expressions involving enums.